### PR TITLE
Check module names in action plugin without collection attached

### PR DESCRIPTION
**Pull request by Qalthos**
##### SUMMARY

Like #60446, but for the other supported platforms

##### ISSUE TYPE

- Bugfix Pull Request

##### COMPONENT NAME

eos_config

Depends-On: https://github.com/ansible/ansible/pull/61605
Depends-On: https://github.com/ansible/ansible/pull/61592

### CODE REVIEW

File: lib/ansible/plugins/action/junos.py [hunk 1]
```diff
@@ -48,12 +49,12 @@ def run(self, tmp=None, task_vars=None):
             pc.network_os = 'junos'
             pc.remote_addr = provider['host'] or self._play_context.remote_addr
 
-            if provider['transport'] == 'cli' and self._task.action not in CLI_SUPPORTED_MODULES:
+            if provider['transport'] == 'cli' and module_name in CLI_SUPPORTED_MODULES:
```
Comment (Qalthos) [inline lib/ansible/plugins/action/junos.py]:
```suggestion
            if provider['transport'] == 'cli' and module_name not in CLI_SUPPORTED_MODULES:
```

**Review by Qalthos — COMMENTED** [review]
Whoops

**Comment by Qalthos** [comment]
recheck

**Comment by pabelanger** [comment]
recheck

**Comment by pabelanger** [comment]
https://logs.zuul.ansible.com/8/8/fcf899a966a83c7461dd365c9b262a6f919eb326/check/ansible-test-network-integration-junos-netconf-python36/613e699/controller/ara-report/ is an example of this not working for netconf_config I believe.  Is there something specific we also need to ensure for it?

**Comment by pabelanger** [comment]
@Qalthos this looks to be working as expected, however I am unsure if iosxr_bgp failure is related or another issue: https://logs.zuul.ansible.com/15/15/4d74bb5fdbd0a5fdfcb209bc0ebda1e469b1c4f4/check/ansible-test-network-integration-iosxr-python27/28f6711/controller/ara-report/result/e0381346-461a-45f2-b766-f06c7332ef91/

**Comment by pabelanger** [comment]
recheck

**Comment by abadger** [comment]
This looks to be a non-invasive bugfix and can be merged between beta1 and rc1.  not blocking beta1 on it.

**Comment by pabelanger** [comment]
recheck

**Comment by pabelanger** [comment]
recheck

**Comment by pabelanger** [comment]
This fixes our collection issue, but we need to fix tests here.

**Comment by pabelanger** [comment]
closing to re-trigger jobs

**Comment by pabelanger** [comment]
recheck

**Comment by pabelanger** [comment]
recheck

**Comment by pabelanger** [comment]
recheck

**Comment by pabelanger** [comment]
recheck

**Comment by pabelanger** [comment]
The iosxr error above, is new to me, but believe we have an open issue for it.

**Pull request merged**